arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

```diff
@@ -294,4 +294,12 @@
 			/* No configuration till now*/
 		};
 	};
+
+	/* PHY1 (SoC Ethernet) */
+	fragment@8 {
+		target = <&phy1>;
+		__overlay__ {
+			led-modes = <0x08 0x00>; /* Link Speed/Activity */
```

Comment:
Small nit pick, as I dislike the postfix comment style. It is more natural to have the comment above the line. So you read the comment and with this context you can assess the code below. And if you prefer to ignore commits you just can skip the line.